socialhome/streams/templates/streams/vue.html

```diff
@@ -0,0 +1,62 @@
+{% extends "streams/base.html" %}
+{% load staticfiles json_safe string_utils %}
+
+{% block content %}
+    {# Temporary hack; to be removed #}
+    <script>
+        window.templateData = {
+            contentList: [
+                {% for content in content_list %}
+                    {
+                        htmlSafe: {{ content.rendered|json_safe }},
+                        author: {
+                            guid: "{{ content.author.guid }}",
+                            name: "{% if content.author.name %}{{ content.author.name }}{% else %}{{ content.author.handle }}{% endif %}",
+                            handle: "{{ content.author.handle }}",
+                            imageUrlSmall: "{{ content.author.safer_image_url_small }}",
+                            absoluteUrl: "{{ content.author.get_absolute_url }}",
+                            homeUrl: "{{ content.author.home_url }}",
+                            isUserFollowingAuthor: {% if not content.author.id in request.user.profile.following_ids %}true{% else %}false{% endif %}
+                        },
+                        id: {{ content.id }},
+                        timestamp: "{{ content.formatted_timestamp }}",
+                        humanizedTimestamp: "{{ content.humanized_timestamp }}",
+                        edited: {{ content.edited|json_safe }},
+                        childrenCount: {{ content.children.count }},
+                        isUserLocal: {% if content.author.user %}true{% else %}false{% endif %},
+                        contentUrl: "{{ content.get_absolute_url }}",
```

Comment:
Have you rebased your branch on top of master recently? `Content.get_absolute_url` method was only added some days ago, around the time I suggested you to use it ;)